module/main.go

```diff
@@ -51,6 +52,13 @@ func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error
 		}
 	}()
 
+	exporter, err := telemetry.Init()
```

Comment:
only boot this & defer if in log level if it is going to output telemetry info, regardless of whether the application writes to counters or not